### PR TITLE
FORGE-2255: Adding helper methods in CommandController

### PR DESCRIPTION
**Pull request by agoncal**

### CODE REVIEW

File: ui/api/src/main/java/org/jboss/forge/addon/ui/controller/CommandController.java [hunk 1]
```diff
@@ -97,6 +97,20 @@
    Map<String, InputComponent<?, ?>> getInputs();
 
    /**
+    * Returns a {@link InputComponent} instances for this command given an input name.
```
Comment (lincolnthree) [inline ui/api/src/main/java/org/jboss/forge/addon/ui/controller/CommandController.java]: (May be <code>null.</code>)

Comment (lincolnthree) [inline ui/api/src/main/java/org/jboss/forge/addon/ui/controller/CommandController.java]: Might want to add that.

**Comment by lincolnthree** [comment]
Thanks Antonio!

**Comment by gastaldi** [comment]
Thanks Antonio

**Pull request merged**